vault/provider.go

```diff
@@ -553,10 +553,10 @@ var (
 			Resource:      UpdateSchemaResource(rabbitMQSecretBackendRoleResource()),
 			PathInventory: []string{"/rabbitmq/roles/{name}"},
 		},
-		// "vault_password_policy": {
-		// 	Resource:      UpdateSchemaResource(passwordPolicyResource()),
-		// 	PathInventory: []string{"/sys/policy/password/{name}"},
-		// },
+		"vault_password_policy": {
```

Comment:
Uncommented for now to let CI be green 🟢 . This may be removed from the SDKv2 provider once we add tests for the new framework password policy resource and confirm it works as expected